tokenizers/src/pre_tokenizers/split.rs

```diff
@@ -1,4 +1,4 @@
-use onig::Regex;
+use crate::utils::SysRegex as Regex;
```

Comment:
`SysRegex` is used in other places of the code, won't this alias make the code a bit confusing ? Either work for me, `SysRegex` or `Regex`

Reply:
There's a lot of `Regex` everywhere in code. 

`SysRegex` tends to imply `System` meaning the real underlying regex engine.
Happy to put a better name.

And I should rename everything instead of using `as Regex` it was because of laziness :)